lib/libvarnishapi/vxp_lexer.c

```diff
@@ -45,7 +45,7 @@
 #include "vxp.h"
 
 static void
-vxp_add_token(struct vxp *vxp, unsigned tok, const char *b, const char *e)
+vxp_append_token(struct vxp *vxp, unsigned tok, const char *b, const char *e)
```

Comment:
Hehe, I guess this change belonged in the previous patch (ref my comment there)

Reply:
My understanding is that this comment resolves the previous one after a brief discussion.